R/powerpoint_presentation.R

```diff
@@ -54,6 +70,27 @@ powerpoint_presentation <- function(
       args = args
     ),
     keep_md = keep_md,
-    df_print = df_print
+    df_print = df_print,
+    intermediates_generator = intermediates_generator
   )
 }
+
+powerpoint_intermediates_generator <- function(saved_files_dir,
+                                               original_input,
+                                               encoding, intermediates_dir) {
+
+  # copy all intermediates (pandoc will need to bundle them in the Powerpoint)
+  intermediates <- copy_render_intermediates(original_input, encoding,
+                                             intermediates_dir, FALSE)
```

Comment:
So basically this fix relies on `resource_files` in the YAML frontmatter. I hope that we don't have to use this field. Instead, automatically copy the file specified in the `reference_doc` argument to the intermediate dir. @slopp's example could be:

```yaml
output:
  powerpoint_presentation:
    reference_doc: rstudio.pptx
```

instead of

```yaml
output:
  powerpoint_presentation:
    pandoc_args: --reference-doc=./rstudio.pptx
resource_files:
- rstudio.pptx
```

The latest CRAN version of **rmarkdown** has the `reference_doc` argument in `powerpoint_presentation`, so @slopp I recommend that you use this argument instead of `pandoc_args` (which was a workaround I told Nathan quite a few months ago when Pandoc didn't support PPT templates).